tests/Tests/500__jupyterhub/test-jupyterhub-api-telemetry.robot

```diff
@@ -0,0 +1,80 @@
+*** Settings ***
+Resource    ../../Resources/ODS.robot
+Resource    ../../Resources/Page/ODH/JupyterHub/LaunchJupyterHub.robot
+Resource    ../../Resources/Page/ODH/JupyterHub/LoginJupyterHub.robot
+Resource    ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource    ../../Resources/Page/OCPDashboard/OCPMenu.robot
+Library     SeleniumLibrary
+Test Setup    Test Suite For Web
+
+*** Test Cases ***
+Verify Telemetry Data Is Accessible
+    [Tags]  Sanity
+    ...     ODS-347
+    Usage Data Collection Should Be Enabled
+    Load JupyterHub API Page
+    JupyterHub API Should Exposes SegmentKey
+    Switch To RHODS Dashboard
+    Navigate To Cluster Settings Page
+    Disable "Usage Data Collection"
+    Switch To JupyterHub API Tab
+    Reload Page
+    JupyterHub API Should Not Exposes SegmentKey
+    [Teardown]    Teardown For Checks SegmentKey Is Exposes In Jupyterhub API TestCase
+
+*** Keywords ***
+Test Suite For Web
+    Set Library Search Order  SeleniumLibrary
+
+Navigate To Cluster Settings Page
+    [Documentation]     Navigate to Cluste Settings page
+    Menu.Navigate To Page    Settings    Cluster settings
+    Wait Until Page Contains    Usage Data Collection
+
+Teardown For Checks SegmentKey Is Exposes In Jupyterhub API TestCase
+    [Documentation]     Enables ussage data collection and close the browser
+    Switch To RHODS Dashboard
+    Wait Until Page Contains    Usage Data Collection
+    Enable "Usage Data Collection"
+    Usage Data Collection Should Be Enabled
+    Close Browser
+
+Load JupyterHub API Page
+    [Documentation]     Loads Jupyterhub and navigate to API url
+    Open Browser    ${ODH_DASHBOARD_URL}    browser=${BROWSER.NAME}    options=${BROWSER.OPTIONS}
+    Login To RHODS Dashboard    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}
+    Wait for RHODS Dashboard to Load
+    Launch JupyterHub Spawner From Dashboard
+    ${url} =    Get Location
+    Log    ${url}
+    ${url} =    Replace String    ${url}    /hub/spawn    /services/jsp-api/api/instance
+    Go To    ${url}
+    sleep  0.5s
+
+Is API Exposes SegmentKey
```

Comment:
`Is SegmentKey Exposed In JupyterHub API` maybe is a more clear name

Reply:
done